alf/algorithms/actor_critic_loss.py

```diff
@@ -67,6 +85,11 @@ def __init__(self,
             normalize_advantages (bool): If True, normalize advantage to zero
                 mean and unit variance within batch for caculating policy
                 gradient. This is commonly used for PPO.
+            normalize_scalar_advantages (bool): If False, the normalization is
```

Comment:
I think it is not very clear to the reader about the relationship between `normalize_advantages` and `normalize_scalar_advantages`, especially when both of them are `True`. 

In `__init__` it seems that `normalize_scalar_advantages` supersedes `normalize_advantages`, but in `forward` it seems to suggest the other way around.

Reply:
updated dod-string